src/devices_models/devices/hydro_generation.jl

```diff
@@ -435,6 +436,27 @@ function initial_conditions!(
 end
 
 ########################## Addition of to the nodal balances ###############################
+
+#function NodalExpressionInputs(::Type{<:PSY.HydroGen}, ::Type{<:PM.AbstractPowerModel}, use_forecasts::Bool)
```

Comment:
What should we do about this section?  I left the existing code.

Reply:
This code is a copy of the RenewableEnergy and it is non-sensical we can eliminate it